src/style.css

```
@@ -1 +1,34 @@
-/* add styles here */
+.body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+  padding: 0;
+  margin: 0;
+}
+
+.stars__star {
+  background-repeat: no-repeat;
+  background-position: center;
+  display: inline-block;
+
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars__star:nth-child(-n + 5) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+}
```

You can rewrite like this
```suggestion
.stars__star {
  background-repeat: no-repeat;
  background-position: center;
  display: inline-block;

  width: 16px;
  height: 16px;
  margin-right: 4px;

  background-image: url(images/star-active.svg);
  background-repeat: no-repeat;
}
```